.github/workflows/pull-request.yml

```diff
@@ -16,22 +16,31 @@
 name: pull request
 
 on:
-  pull_request:
+  pull_request_target:
     types:
-      - reopened
-      - opened
-      - edited
-      - synchronize
+    - reopened
+    - opened
+    - edited
+    - synchronize
+    - pull_request_target
 
 jobs:
   title:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout project
-        uses: actions/checkout@v4
-      - name: Check pull request title
-        env:
-          TITLE: ${{ github.event.pull_request.title }}
-        run:
-          .github/scripts/pr_title_check.sh "${TITLE}"
-
+    - name: Checkout project
+      uses: actions/checkout@v4
+    - name: Check pull request title
+      env:
+        TITLE: ${{ github.event.pull_request.title }}
+      run:
+        .github/scripts/pr_title_check.sh "${TITLE}"
```

Comment:
GitHub Actions doc [says](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) about `pull_request_target`:

> Avoid using this event if you need to build or run code from the pull request.

Thus, I think we should keep these checks separate.

Reply:
Ah makes sense. Let's keep the scope of this workflow restricted since it requires greater permissions. I'll close this as won't fix.